mmv1/third_party/terraform/website/docs/d/sql_database_instance.html.markdown

```diff
@@ -179,3 +179,13 @@ The `replica_configuration` block contains:
 * `server_ca_cert.0.expiration_time` - Expiration time of the CA Cert.
 
 * `server_ca_cert.0.sha1_fingerprint` - SHA Fingerprint of the CA Cert.
+
+The `settings.insights_config` subblock for instances declares [Query Insights](https://cloud.google.com/sql/docs/postgres/insights-overview) configuration. It contains:
```

Comment:
Can we move this sub block just above the `replica_configuration` block (line 129)? To be closer with other `settings` blocks

Reply:
Agreed https://github.com/GoogleCloudPlatform/magic-modules/pull/4464/commits/8464a2023f375aa7e28ad1400a38b4fcc3b88ed2